e2e/tests/components/select/standard/select.e2e-spec.ts

```diff
@@ -544,20 +546,20 @@ describe('Select Tests', () => {
         await page.toggleCustomIcon();
         await page.clickOnDropdown(false);
         await page.clickOnCountry(false, 1);
-        expect(await customIcon.isPresent()).toBeTruthy();
+        browser.wait(EC.presenceOf(customIcon), 5000);
```

Comment:
I think we should probably have a constant for the wait value, hardcoded values are a pain to update.